services/keybase/keybase-btc.service.js

```diff
@@ -0,0 +1,111 @@
+'use strict'
+
+const KeybaseProfile = require('./keybase-profile')
+const Joi = require('joi')
+const { nonNegativeInteger } = require('../validators')
+
+const bitcoinAddressSchema = Joi.object({
+  status: Joi.object({
+    code: nonNegativeInteger.required(),
+  }).required(),
+  them: Joi.array()
+    .items(
+      Joi.object({
+        cryptocurrency_addresses: Joi.object({
+          bitcoin: Joi.array().items(
+            Joi.object({
+              address: Joi.string().required(),
+            }).required()
+          ),
+        })
+          .required()
+          .allow(null),
+      })
+        .required()
+        .allow(null)
+    )
+    .min(0)
+    .max(1),
+}).required()
+
+module.exports = class KeybaseBTC extends KeybaseProfile {
+  static get apiVersion() {
+    return '1.0'
+  }
+
+  static get category() {
```

Comment:
Since we're going to use the same category, we can move this function to the base class

Reply:
Make sense. Now you say it, I don't think we will one day have new Keybase badges in another category.